FitNesseRoot/DbFit/AcceptanceTests/JavaTests/TeradataTests/FlowMode/SetUp/content.txt

```diff
@@ -1,7 +1,9 @@
 !|dbfit.TeradataTest|
 
+|Connect Using File|!-TestDbConnectionDbFitTeradata.properties-!|
```

Comment:
FitNesse also passes the modified word through to Fit:-

![image](https://user-images.githubusercontent.com/1895701/27725276-7cc5fa6e-5d6d-11e7-92c4-c0ba509525de.png)